### PR TITLE
Solver -- Replacing GLPK With CBC

### PR DESCRIPTION
**Pull request by jeisenman23**
This PR removes the GLPK solver and replaces it with the CBC solver.

### CODE REVIEW

**Comment by jgu2** [comment]
Great work, CBC solver testing works at my local.

One more thing:

Update the Sphinx documentation as well, remove GLPK info and add CBC.

@jeisenman23

File: calliope_app/compose/Dockerfile [hunk 1]
```diff
@@ -19,22 +19,16 @@ RUN apt-get update -y --fix-missing \
     gettext \
     postgresql-client \
     wget \
+    curl \
```
[the remark this answers is not in the document]
Comment (jeisenman23) [reply calliope_app/compose/Dockerfile]: Resolved

**Comment by jeisenman23** [comment]
> Great work, CBC solver testing works at my local.
> 
> One more thing:
> 
> Update the Sphinx documentation as well, remove GLPK info and add CBC. - https://github.com/NREL/engage/tree/dev/docs/sources
> 
> @jeisenman23

Done

File: calliope_app/api/fixtures/sample_scenario_param.json [hunk 2]
```diff
@@ -32,16 +32,6 @@
   {
     "pk": 4,
     "model": "api.scenario_param",
-    "fields": {
-      "scenario_id": "1",
-      "run_parameter_id": "23",
-      "value": "glpk",
```
Comment (jgu2) [inline calliope_app/api/fixtures/sample_scenario_param.json:38]: It looks like this record got deleted.

Would this `glpk` be changed to `cbc`?

Rename the following `pk` number back.

**Comment by jgu2** [comment]
calliope_app.api.fixtures.admin_run_parameters.json, rename `glpk` to `cbc`.

**Review by jgu2 — CHANGES_REQUESTED** [review]
Previous work got lost when doing PR merges, need to check the logs and revert back.

**Pull request merged**